src/inferTsconfig.ts

```diff
@@ -19,7 +19,7 @@ export const noJsConfig = '{}'
  * If the directory contains at least one `*.{ts,tsx}` file then the config will be empty (`{}`).
  * If the directory doesn't contains one `*.{ts,tsx}` file then the config will
  */
-export function inferTsConfig(projectPath: string): string {
+export function inferTsconfig(projectPath: string): string {
```

Comment:
I apologize for mixing this unrelated diff into the PR. It's quite painful to correct the capitalization of files on macOS and I didn't feel like doing it again to have a cleaner PR